README.md

```diff
@@ -13,11 +13,11 @@ N.B. Make sure to install requirements via `pip`, not `conda` (at the time of wr
 ## Running the validator
 
 ```sh
-python tools/validator.py coverage.covjson
+python tools/validator.py my.covjson
```

Comment:
```suggestion
python tools/validator.py my.covjson
```
```suggestion
python -m tools.validator my.covjson
```

`validator.py` doesn't use relative imports, but `bundle_schema.py` now does, and the only consistent way to run scripts with relative imports is to use `python -m` and treat them as modules.  The same change has to be made in `.github\workflows\publish-schema.yml` when invoking the `tools/` scripts. That workflow only runs after merging, on `main`.